service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class HearingBookingService {
+
+    public List<Element<HearingBooking>> expandHearingBookingCollection(CaseData caseData) {
+        if (caseData.getHearingDetails() == null) {
+            List<Element<HearingBooking>> populatedHearing = new ArrayList<>();
```

Comment:
return ImmutableList.of(Element.<HearingBooking>builder()
    .value(HearingBooking.builder().build()))
    .build());

do you not need to set a value for it to expand as well? I seem to remember seeing this in other places

Reply:
No it expands and works like this